py/sea/commands.py

```diff
@@ -899,6 +899,8 @@ def mk_arg_parser (self, ap):
                         metavar='STR', help='Log level')
         ap.add_argument('--no-fat-fns', dest='no_fat_fns', default=None,
                         type=str, metavar='STR,STR,...', help='List of functions to NOT instrument')
+        ap.add_argument ('--enable-fat-vacuity-check', dest='enable_fat_vacuity_check', default=False,
```

Comment:
we have special handling of boolean options. use that.

Reply:
Do you mean we add options into `sea.yaml`?

Reply:
no. how bool options are added to args. there is a special function for that

Reply:
Made it. Used `add_bool_argument`.